src/storage/trie/merkle_trie.rs

```diff
@@ -159,18 +169,32 @@ impl MerkleTrie {
      *  Note: We require a write-locked root node to perform this operation, which should
      *  be supplied by the caller.
      */
-    fn unload_from_memory(&self, root: &mut TrieNode, force: bool) -> Result<(), HubError> {
```

Comment:
just doing some refactoring here to remove this force: flag since it was always `true`